### PR TITLE
damask: add version 3.0.0-alpha8

### PR DESCRIPTION
**Pull request by MarDiehl**
added support for newest version and tested locally

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-damask/package.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ class PyDamask(PythonPackage):
 
     maintainers("MarDiehl")
 
+    version(
+        "3.0.0-alpha8", sha256="f62c38123213d1c1fe2eb8910b0ffbdc1cac56273c2520f3b64a553363190b9d"
+    )
     version(
         "3.0.0-alpha7", sha256="442b06b824441293e72ff91b211a555c5d497aedf62be1c4332c426558b848a4"
     )
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-damask/package.py:23]: Dependencies have specific versions required, at least in the latest version:
```
install_requires =
    pandas>=0.24                                                                                    # requires numpy
    numpy>=1.17                                                                                     # needed for default_rng
    scipy>=1.2
    h5py>=2.9                                                                                       # requires numpy
    vtk>=8.1
    matplotlib>=3.0                                                                                 # requires numpy, pillow
    pyyaml>=3.12
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]
Only reviewed the Python bits.

File: var/spack/repos/builtin/packages/py-damask/package.py (hunk 1 above)
Comment (MarDiehl) [reply var/spack/repos/builtin/packages/py-damask/package.py:23]: @adamjstewart: thanks!

**Review by adamjstewart — APPROVED** [review]
Python bits LGTM, will let someone else review the rest.

**Review by naromero77 — APPROVED** [review]
LGTM

**Pull request merged**